package.json

```diff
@@ -7,9 +7,11 @@
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
-    "lint": "eslint --fix src"
+    "lint": "tslint --config ./tslint.json --project ./tsconfig.json 'src/**/*.ts'",
+    "copy-env": "if ! [ -e .env ]; then cp .env.tpl .env; fi;"
   },
   "dependencies": {
+    "@types/react": "^16.8.22",
```

Comment:
1. Shouldn't `@types` be a dev dependency?
2. Please target all dependencies by strict versions